src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -92,44 +93,81 @@ const NavHeader = (props) => {
 
   return (
-    <Animated.View style={styles.container(animatedHeight)}>
+    <View style={{}}>
       <StatusBar barStyle="dark-content" />
       <View
-        source={{uri: coverPath}}
-        style={[styles.navContainer(isHeaderHide, topPosition), heightWithCoverImage()]}
+        style={[
+          styles.navContainer(isHeaderHide),
+          {
+            height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+          }
+        ]}
         imageStyle={{opacity: isHeaderHide ? 0 : 1}}>
-        <Animated.View style={styles.headerImage(opacityHeaderAnimation)}>
-          <FastImage source={{uri: coverPath}} style={heightWithCoverImage()} />
-        </Animated.View>
-        <TouchableOpacity onPress={() => backScreen()} style={styles.backbutton}>
-          {coverPath && !isHeaderHide ? (
-            <MemoIcArrowBackCircle width={normalize(32)} height={normalize(32)} />
-          ) : (
-            <MemoIcArrowBack width={normalize(24)} height={normalize(24)} />
-          )}
-        </TouchableOpacity>
-        <View style={styles.containerAction}>
+        {isLoading ? (
+          <Shimmer width={displayWidth} height={dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER} />
+        ) : (
+          <>
+            <View
+              style={{
+                width: '100%',
+                height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+              }}>
+              <FastImage
+                source={{uri: coverPath}}
+                style={{
+                  height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+                }}
+              />
+              <AnimatedBlurView
+                style={[
+                  StyleSheet.absoluteFillObject,
+                  {
+                    opacity: opacityImage
+                  }
+                ]}
+                blurType="light"
+              />
+            </View>
+          </>
+        )}
+
+        <View
+          style={{
+            flexDirection: 'row',
+            alignItems: 'center',
+            paddingTop: insets.top,
+            zIndex: 2,
+            position: 'absolute'
+          }}>
+          <TouchableOpacity onPress={() => backScreen()} style={styles.backbutton}>
+            {coverPath && !isHeaderHide ? (
+              <MemoIcArrowBackCircle width={normalize(32)} height={normalize(32)} />
+            ) : (
+              <MemoIcArrowBack width={normalize(24)} height={normalize(24)} />
+            )}
+          </TouchableOpacity>
+          <Animated.View
+            style={{
+              opacity: opacityImage
+            }}>
+            <TopicDomainHeader {...props} />
+          </Animated.View>
+        </View>
+
+        <View
+          style={
+            (styles.containerAction,
+            {
+              paddingTop: insets.top,
+              paddingRight: dimen.normalizeDimen(20),
+              zIndex: 2,
+              position: 'absolute',
+              right: 0
+            })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In line 100, you have added a `StatusBar` component with `barStyle="dark-content"`. This will set the status bar text color to dark. Make sure this change is consistent with the rest of your app's design. 

Also, in line 108, you are using conditional rendering based on the `isLoading` prop. This is a good practice as it improves user experience by providing feedback during data loading. However, ensure that the `Shimmer` component provides an accurate representation of the content being loaded.



<!-- This is an auto-generated comment by OSS CodeRabbit -->